packages/manager/modules/nutanix/src/dashboard/general-info/redeploy/template.html

```diff
@@ -254,37 +253,28 @@ <h2 data-translate="nutanix_redeploy_title"></h2>
                             data-oui-popover="{{:: 'nutanix_redeploy_prism_central_ip_description' | translate}}"
                         ></button>
                     </div>
-                    <oui-button
-                        class="ml-1"
-                        data-size="s"
-                        data-variant="primary"
-                        data-on-click="$ctrl.addPrismCentralIp()"
-                    >
-                        <span
-                            data-translate="nutanix_redeploy_custom_configuration_add_prism_central_ip"
-                        ></span>
-                    </oui-button>
                     <div
                         class="row ml-1 mt-2"
                         data-ng-repeat="item in $ctrl.config.prismCentral.ips track by $index"
                     >
-                        <input
-                            type="text"
-                            class="oui-input w-50"
-                            name="prismCentralIps{{$index}}"
-                            data-ng-model="$ctrl.config.prismCentral.ips[$index]"
-                            required
-                            data-ng-pattern="$ctrl.IPV4_REGEX"
-                        />
-                        <oui-button
-                            class="ml-2"
-                            data-size="s"
-                            data-variant="secondary"
-                            data-on-click="$ctrl.removePrismCentralIp($index)"
-                        >
-                            <span class="oui-icon oui-icon-bin"></span>
-                        </oui-button>
+                        <form name="ipForm{{$index}}" novalidate class="w-50">
```

Comment:
**nitpick**
If possible, could be interesting to track by ips (if they are unique), and then use the ip to define unique form names and input names

Reply:
The list of IPs are maintained as an array of strings and they are empty on page load. So they are not unique and cannot be used in the form name.